examples/platforms/efr32mg21/radio.c

```diff
@@ -696,10 +660,6 @@ static void processNextRxPacket(otInstance *aInstance, RAIL_Handle_t aRailHandle
 
         sReceiveError = OT_ERROR_NONE;
 
-        // TODO Set this flag only when the packet is really acknowledged with frame pending set.
-        // See https://github.com/openthread/openthread/pull/3785
-        sReceiveFrame.mInfo.mRxInfo.mAckedWithFramePending = true;
```

Comment:
Why remove line 701?

Reply:
It is still present, the function has moved, check line num 686.